lib/compress/zstd_compress.c

```diff
@@ -4421,7 +4445,8 @@ static size_t ZSTD_loadZstdDictionary(ZSTD_compressedBlockState_t* bs,
                                       ZSTD_CCtx_params const* params,
                                       const void* dict, size_t dictSize,
                                       ZSTD_dictTableLoadMethod_e dtlm,
-                                      void* workspace)
+                                      void* workspace,
+                                      ZSTD_tableFillPurpose_e tfp)
```

Comment:
This arg is probably more at home next to `dtlm`, rather than having `workspace` between them?